src/registry/children/ChildrenTable.tsx

```diff
@@ -0,0 +1,167 @@
+      )}
+    >
+      <UtrechtIcon>
+        <Icon icon="edit" />
+      </UtrechtIcon>
+    </SubtleButton>
+  );
+};
+
+interface ChildDeleteButtonProps {
+  child: ExtendedChildDetails;
+  removeChild: () => void;
+}
+
+const ChildDeleteButton: React.FC<ChildDeleteButtonProps> = ({child, removeChild}) => {
+  const intl = useIntl();
+  return (
+    <SubtleButton
+      onClick={removeChild}
+      hint="danger"
+      aria-label={intl.formatMessage(
+        {
+          description: "Children component: children table 'delete child' accessible label",
+          defaultMessage: 'Delete child with BSN: {bsn}',
+        },
+        {bsn: child.bsn}
+      )}
+    >
+      <UtrechtIcon>
+        <Icon icon="remove" />
+      </UtrechtIcon>
+    </SubtleButton>
+  );
+};
+
+const EmptyMessage: React.FC = () => (
+  <i>
+    <FormattedMessage description="Emtpy field message" defaultMessage="No information provided" />
+  </i>
+);
+
+export interface ChildrenTableProps {
+  values: ExtendedChildDetails[];
+  enableSelection?: boolean;
+  editChild: (child: ExtendedChildDetails | undefined) => void;
+  removeChild: (childIndex: number) => void;
+  selectChild: (childIndex: number, child: ExtendedChildDetails) => void;
+}
+
+const ChildrenTable: React.FC<ChildrenTableProps> = ({
+  values,
+  enableSelection,
+  editChild,
+  removeChild,
+  selectChild,
+}) => {
+  const intl = useIntl();
+  return (
+    <Table>
+      <TableHeader>
+        <TableRow>
+          {enableSelection && <TableHeaderCell scope="col" />}
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'bsn' header"
+              defaultMessage="BSN"
+            />
+          </TableHeaderCell>
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'firstNames' header"
+              defaultMessage="First names"
+            />
+          </TableHeaderCell>
+          <TableHeaderCell scope="col">
+            <FormattedMessage
+              description="Children component: children table 'dateOfBirth' header"
+              defaultMessage="Date of birth"
+            />
+          </TableHeaderCell>
+        </TableRow>
+      </TableHeader>
+      <TableBody>
+        {values.map((child, index) => (
+          <TableRow key={child.__id || index}>
+            {enableSelection && (
+              <TableCell>
+                <UtrechtCheckbox
+                  name="selected"
+                  aria-label={intl.formatMessage(
+                    {
+                      description: "Children component: child 'selected' aria label",
+                      defaultMessage: '{firstname} should be included in the form submission',
+                    },
+                    {
+                      firstname: child.firstNames,
+                    }
+                  )}
+                  checked={child.selected}
```

Comment:
Okay good that there is a label, I don't think this is the optimal label text yet though.

```suggestion
                  aria-label={intl.formatMessage(
                    {
                      description: "Children component: child 'selected' aria label",
                      defaultMessage: 'Include {firstname}?',
                    },
                    {
                      firstname: child.firstNames,
                    }
                  )}
                  checked={child.selected}
```

I don't think the user needs to be bothered with "form submission" concepts.

Reply:
Mhh, yeah thats better. Even just wording it as a question, makes it a lot better